app/controllers/welcome_controller.rb

```diff
@@ -56,6 +56,8 @@ def authenticate_user_for_welcome_controller
   # Split the given array `list` into two halves
   # Return a tuple with each half as an array.
   def list_halves(list)
+    return [[], []] if list.empty?
```

Comment:
Would you add a test for the bug?

Reply:
Added -- https://github.com/bikeindex/bike_index/pull/710/commits/d04c8bb61d3c269da6796e5b019ea4a0d024b165#diff-50d29b5fa2c03989d9bc2490d16e6b9bR156